pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -842,6 +849,31 @@ def _report_success(self):
 
         jenkins.update_description(f'<hr />Build results:<br/><br/>{timing_report}<br/>')
 
+    def _mass_rebuild_score(self) -> int:
+        """For the ocp_version (e.g. `4.16`) return an integer score value
+        Higher the score, higher the priority
+        """
+        return round(float(self.version.stream) * 100)  # '4.16' -> 416
+
+    async def _request_mass_rebuild(self):
+        coro = self._rebase_and_build_images()
+        ocp_version = self.version.stream
+        lock_id = self.lock_identifier
+        lock = Lock.MASS_REBUILD
+        lock_name = Lock.MASS_REBUILD.value
+        queue = locks.Keys.MASS_REBUILD_QUEUE.value
+        mapping = {ocp_version: self._mass_rebuild_score()}
+
+        # add yourself to the queue
+        # if queue does not exist, it will be created with the value
+        await redis.call('zadd', queue, mapping)
```

Comment:
```suggestion
        await redis.call('zadd', locks.Keys.MASS_REBUILD_QUEUE.value, {self.version.stream: self._mass_rebuild_score()})
```

Reply:
I would like to keep this for readability